react-frontend/src/login/InformativeTextComponent.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Row } from "core-components/grid/GridComponent";
+
+const HrLine = styled.hr`
+  width: 10%;
+  background: white;
+`;
+
+const H3 = styled.h3`
+  margin: 2% 20% 0%;
+`;
+
+const InformativeTextComponent = ({
+  className,
+  informativeText,
+  encouragementThought,
+}) => (
+  <Row className={className}>
```

Comment:
Please decide whether you want a row or a column. You have added classes for both "row" and "flex-column".

If there is a rational behind it, please explain.

CC: @mayuriardad